pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md

```diff
@@ -12,7 +12,8 @@ render: mustache
 ## Updates
 
 - Update to Confluent Kafka version 4.1.2
-- Upgrade to sdk 0.55.2
+- Update dcos-coommons sdk version to 0.55.2
+- Update Zookeeper Client version to 3.4.13
```

Comment:
Can you also a note that we are switching to file-based secrets here?

Reply:
Done